view/embed-code.php

```diff
@@ -9,6 +9,7 @@
 $width          = $embedOptions['width'];
 $height         = $embedOptions['height'];
 $isResponsive   = !empty($embedOptions['responsive']);
+$hoveringControls = $embedOptions['hoveringControls'] === 'true';
```

Comment:
If `hoveringControls` is handled as `responsive` attribute, then the condition should be the same here (`!empty($embedOptions['hoveringControls']);`)